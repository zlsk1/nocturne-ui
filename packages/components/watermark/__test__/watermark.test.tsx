import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import Watermark from '../src/watermark.vue'

describe('Watermark.vue', () => {
  it('render', () => {
    const wrapper = mount(() => (
      <Watermark class="watermark" content="Noc">
        watermark
      </Watermark>
    ))

    expect(wrapper.classes()).toContain('watermark')
    expect(wrapper.html()).toMatchSnapshot()
  })
})
