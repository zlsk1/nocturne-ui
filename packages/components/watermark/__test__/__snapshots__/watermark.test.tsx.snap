// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Watermark.vue > render 1`] = `"<div style="position: relative; overflow: hidden;" class="watermark">watermark</div>"`;
